tests/actor/test_run_utils.py

```diff
@@ -0,0 +1,112 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Simple tests for the actor runner."""
+
+import time
+from typing import Iterator
+from unittest.mock import patch
+
+import async_solipsism
+import pytest
+import time_machine
+from async_solipsism.socket import asyncio
```

Comment:
I guess this was a wrong completion by "smart completion"? Shouldn't this be `import asyncio`?

Reply:
done and fixed the import in another test in a separate commit